docs/docs/usage-guide/additional_configurations.md

```diff
@@ -162,15 +162,22 @@ To use Amazon Bedrock and its foundational models, add the below configuration:
 
 ``` 
 [config] # in configuration.toml
-model = "anthropic.claude-v2"
-fallback_models="anthropic.claude-instant-v1"
+model="bedrock/anthropic.claude-3-sonnet-20240229-v1:0"
+model_turbo="bedrock/anthropic.claude-3-sonnet-20240229-v1:0"
+fallback_models=["bedrock/anthropic.claude-v2:1"]
 
 [aws] # in .secrets.toml
 bedrock_region = "us-east-1"
 ```
 
 Note that you have to add access to foundational models before using them. Please refer to [this document](https://docs.aws.amazon.com/bedrock/latest/userguide/setting-up.html) for more details.
 
+If you are using the claude-3 model, please configure the following settings as there are parameters incompatible with claude-3.
```

Comment:
bedrock does not support `temperature` parameter

https://github.com/BerriAI/litellm/blob/1b93ada827c652eed5f4d8ac05b222b10222b80d/litellm/llms/bedrock.py#L79-L87

Reply:
Discussion:
Another approach could be to set the temperature only if the model is bedrock's claude3 within the chat_completion function.